frontend/src/app/containers/CreateCoverPage/step2.tsx

```diff
@@ -1,7 +1,151 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Song } from 'utils/urls';
 
 export type Props = {};
 
-export default function CreateCoverPreviewPage(props: Props) {
-  return <div data-testid="CreateCoverPreviewPage">CreateCoverPreviewPage</div>;
+export default function CreateCoverInfoPage(props: Props) {
+  const history = useHistory();
+  const [Form, setForm] = useState({
+    title: '',
+    category: '0',
+    instrumentType: '',
+    tags: [],
+    description: '',
+  });
+
+  const categories = ['Pop', 'Rock', 'Hip-hop', 'Jazz'];
+
+  const onChangeForm = (
+    e: React.FormEvent<
+      HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement
+    >,
+    key: string,
+  ) => {
+    setForm({ ...Form, [key]: e.currentTarget.value });
+  };
+
+  const onSubmitForm = (e: React.FormEvent) => {
+    e.preventDefault();
+    console.log(Form);
+    history.push(Song(0));
```

Comment:
이 부분은 혹시 Cover Page로 넘겨야 하는 것 아닐까요
```suggestion
    history.push(Cover(0));
```

Reply:
넵 맞습니다. 아직 커버페이지 구현이 안 되어 있어서, 일단 음악 페이지로 넘어갑니다.